Project/view/projectSurvey.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+include('Project/include/headerPage.php');
+
+$Errors = array();
+if (isset($_REQUEST['formSubmitButton'])) {
+	//var_dump($_REQUEST);
+	foreach($_REQUEST['Answers'] as $index => $value){
+		echo "Question $index: $value<br/>";
```

Comment:
You should probably remove this. It looks like debug code, and it's not great to echo stuff before doing a redirect like you're doing below.